mmv1/third_party/terraform/envvar/envvar_utils.go

```diff
@@ -146,6 +150,11 @@ func GetTestUniverseDomainFromEnv(t *testing.T) string {
 	return transport_tpg.MultiEnvSearch(UniverseDomainEnvVars)
 }
 
+// Project Prefix of different universes
+func GetProjectPrefixFromEnv() string {
```

Comment:
In a followup PR could you change the env var to include "Universe" in the namespace? i.e. `GOOGLE_UNIVERSE_PROJECT_PREFIX` / `GetUniverseProjectPrefixFromEnv`

Reply:
Will do